### PR TITLE
Corrected error, stylistic changes

### PR DESCRIPTION
**Pull request by rpetrusha**
## Corrected error, stylistic changes

In reviewing a token replacement, I noticed that this topic was stylistically awkward and that some descriptions were inaccurate.

### CODE REVIEW

File: xml/System.Windows/TextMarkerStyle.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
    
   
 ## Examples  
- The following code example shows how to create a <xref:System.Windows.TextMarkerStyle>.  
+ The following example shows how to create a <xref:System.Windows.TextMarkerStyle>.
```
Comment (mairaw) [inline xml/System.Windows/TextMarkerStyle.xml]: nit: end with colon?

File: xml/System.Windows/TextMarkerStyle.xml [hunk 2]
```diff
@@ -143,7 +143,7 @@
       </ReturnValue>
       <MemberValue>5</MemberValue>
       <Docs>
-        <summary>A lowercase Roman numeral is displayed, starting with the numeral i, for example, i, ii, iii, and iv. The <see cref="T:System.Windows.TextMarkerStyle" /> is automatically incremented for each item added to the list.</summary>
+        <summary>A lowercase Roman numeral starting with the numeral i. For example, i, ii, iii, and iv. The numeral is automatically incremented for each item added to the list.</summary>
```
Comment (mairaw) [inline xml/System.Windows/TextMarkerStyle.xml]: you added is displayed to some summaries and removed from others. By design?

Comment (rpetrusha) [reply xml/System.Windows/TextMarkerStyle.xml]: I'd meant to remove all occurrences of "is displayed." Bugt I don't see any  remaining, @mairaw.

Comment (mairaw) [reply xml/System.Windows/TextMarkerStyle.xml]: Duh. My brain somehow was seeing some changes in reverse. Yes, you're right @rpetrusha.

Comment (rpetrusha) [reply xml/System.Windows/TextMarkerStyle.xml]: That's OK -- I've done it myself more than once, @mairaw.

**Pull request merged**